src/Storefront/Resources/app/storefront/src/plugin/slider/helper/slider-settings.helper.js

```diff
@@ -18,6 +18,10 @@ export default class SliderSettingsHelper {
         const viewportWidth = window.breakpoints[viewport.toLowerCase()];
         const selectedViewportSettings = responsiveSettings[viewportWidth];
 
+        if (!selectedViewportSettings) {
```

Comment:
Hi @runelaenen, thanks for your contribution. 👍  Just to clarify: The problem is that `window.breakpoints[viewport.toLowerCase()]` returns no result and not the `responsiveSettings` from `options.responsive`?

Does this only happen in the most recent 6.4.4.0 version?

Reply:
Yes, it only started happening after the update.